packages/csharp/graphschema/PropertyNames.cs

```diff
@@ -0,0 +1,157 @@
+// Code generated by Cuelang code gen. DO NOT EDIT!
+// Cuelang source: github.com/specterops/bloodhound/-/tree/main/packages/cue/schemas/
+namespace SharpHoundCommon.Enums {
```

Comment:
For Go/TS we have linters that help with spacing and other readability bits. Is there something similar we could set up to run on this file?

Reply:
Fair enough